packages/experimental/Button/src/useButton.ts

```diff
@@ -18,11 +18,12 @@ export const useButton = (props: ButtonProps): ButtonState => {
       accessibilityRole: 'button',
       onAccessibilityTap: props.onAccessibilityTap || props.onClick,
       accessibilityLabel: props.accessibilityLabel || props.content,
-      accessibilityState: getAccessibilityState(!!disabled),
+      accessibilityState: getAccessibilityState(!!disabled || !!loading),
```

Comment:
Is loading supposed to make the button color disabled while showing? If so you should edit buttonLookup in Button.tsx to apply the disabled state if loading is true.

Reply:
It should be disabled, but shouldn't look like disabled :) Here's how it should look: 
![image](https://user-images.githubusercontent.com/11574680/144579900-07ec1be0-1dbd-4eae-b3f9-01688a9d0824.png)